spec/unit/recipes/_restart_marker_spec.rb

```diff
@@ -0,0 +1,18 @@
+# coding: UTF-8
+
+require_relative '../spec_helper'
+
+describe 'cerner_splunk::_restart_marker' do
+  subject do
+    runner = ChefSpec::SoloRunner.new
+    runner.converge(described_recipe)
+  end
+
+  before do
+    allow(CernerSplunk).to receive(:restart_marker_file).and_return('/opt/splunk')
```

Comment:
I see what you're doing here and it doesn't matter too much but '/opt/splunk' is more than likely the install directory, the marker file is more likely actually '/var/chef/cache/.restart_splunk' 

It could be style, but I would want to either have mock values that are either likely real values, or values (to someone who doesn't know much about Splunk installation) that are very obviously wrong as I don't want tests to add confusion.